core/src/main/java/org/apache/accumulo/core/conf/AccumuloConfiguration.java

```diff
@@ -209,13 +209,19 @@ static public long getMemoryInBytes(String str) {
         case 'B':
           multiplier = 0;
           break;
+        case '%':
```

Comment:
Could have a new type `BYTES` which has everything but the `%`.  And `MEMORY` could be what `BYTES` supports plus `%`.